script/Arcueid/Day 7/07_00_ARC07_1D.txt

```diff
@@ -117,6 +117,7 @@ But a long time ago, I was taught about what was the most righteous choice.
 [2446bf2f4bb36c26c8241ea13aa4992c468b968f]{
 -- Page 119, Offset 16874.
 -- 何もしないって選択だけは、しちゃいけない」
+// 何もしないって選択だけは、しちゃいけない」
```

Comment:
you can remove this duplicate comment. I'm guessing this is from a script Ross wrote to copy in script comments from old files.